src/cascadia/TerminalApp/AppLogic.idl

```diff
@@ -60,6 +60,7 @@ namespace TerminalApp
         void Quit();
 
         void LoadSettings();
+        void ReloadSettings();

```

Comment:
Having both of these moves the burden of knowing when to use which up to the caller. Can they be consolidated? Can `ReloadSettings` just implicitly do a `LoadSettings` if it's the first one?